### DISCUSSION TITLE
Bilişim Yaz ödevleri

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#314] [Bilsem]
2022/2023 Öğretim döneminden sonra yaz ödevleriniz aşağıda listelenmiştir. Bütün bilişim öğrencilerinin bu ödevleri yapması gerekiyor.

Bütün ÖYG ve Proje öğrencilerinin github hesabı olması gerekiyor. (Eğer bu güne kadar açmadıysanız veya hesabınıza ulaşamıyorsanız bu sorunu halledin.)

[sonsuzus tartışma ](https://github.com/sonsuzus/sonsuzus/discussions)  bölümündeki sorulardan her öğrenci daha önce cevaplanmamış 5 farklı soruyu çözecek. (Bunu ne kadar erken yaparsanız o kadar iyi çünkü başkaları çözdükçe size zor sorular kalacaktır) 

[Bilsem bilişim proje fikirleri](https://github.com/sonsuzus/sonsuzus/discussions/313) başlığına, açıklamalara uygun olarak en az 10 proje fikri yazılacak.

[sonsuzus.github.io](https://sonsuzus.github.io/) adresindeki [python](https://sonsuzus.github.io/categories/python/) ile ilgili konular okunacak, okuduğunuz tüm konulara ifade bırakın, ayrıca anlamadığınız kısımları konunun altındaki yorum bölümüne yazın.

[hackerearth.com](https://www.hackerearth.com) sitesine üye olunup, daha önce eklenmeyen 5 soruyu Türkçeye çevirip [tartışmalar ](https://github.com/sonsuzus/sonsuzus/discussions) bölümünde Soru&Cevap kısmına ekleyeceksiniz. (Bu konuda da ne kadar hızlı olursanız o kadar iyi) Problemi Türkçeye çevirirken okunaklı anlaşılır ve doğru biçimde eklediğinize emin olun, soru linkini de eklemeyi unutmayın. Tabiki çözümünüzü de yazabilirsiniz altına yorum olarak. Örnek #315 

[github/sonsuzus](https://github.com/sonsuzus/sonsuzus) ve [tartışmalar](https://github.com/sonsuzus/sonsuzus/discussions) bölümlerini yıldızlayın, favorileyin, takibe alın. Denk geldiğiniz bütün gönderileri beğenin, ifade bırakın, bolca yorum yazın.

Buraya kadar olanlar yapmanız zorunlu olan ödevleriniz. Bunun haricinde

- İngilizcesi iyi ve çeviri konusunda yardım edebilecek
- Mitoloji konusunda yazarlık yapmak isteyen
- Programlama konusunda içerik üretebilecek

kişiler de özelden yazabilir bana.

**Comment by slos8888** [comment]
Hocam başkalarının yazdıkları soruları çözsek de sayılır mı?

**Reply by sonsuzus** [reply]
Sayılır.

**Comment by Sevvalgocgun1** [comment]
Hocam çözmemiz gereken sorular bende acilmayip hata veriyor ne yapmalıyım?

**Reply by sonsuzus** [reply]
Hangi sorular hata veriyor? Buraya yazabildiğine göre görebiliyor olman lazım onları da.
